doc/can-route.md

```diff
@@ -140,21 +153,24 @@ You can listen to changes in the url by listening on the underlying route data.
 your route data and rule might have a page property:
 
 ```js
+import {DefineMap, route} from "can";
 const AppViewModel = DefineMap.extend( {
```

Comment:
no need to define a custom type, just use `route.data`.

Reply:
- [x] using `route.data`.